### ISSUE TITLE
Not compatible with Wordpress 5 (and it's new Editor)

### ISSUE THREAD

**Issue by tarator** [qTranslate-Team/qtranslate-x#606]
Are there any plans in migrating this plugin to Wordpress 5 yet?

**Comment by tarator** [comment]
This fork might eventually be updated: https://github.com/qtranslate/qtranslate-xt

**Comment by herrvigg** [comment]
Yes in qTranslate-XT we support WP5 with the Classic Editor. Not the Block Editor yet but it might come some day.

Note qTranslate-X is abandoned since 2016 so you won't get much help here ;)

**Comment by davidjbrossard** [comment]
How easy is it to migrate from QTranslate-X?
